src/server/ocpi/ocpi-services-impl/ocpi-2.1.1/EMSPLocationsEndpoint.ts

```diff
@@ -76,7 +76,7 @@ export default class EMSPLocationsEndpoint extends AbstractEndpoint {
     }
     if (evseUid) {
       const chargingStationId = OCPIUtils.buildChargingStationId(locationId, evseUid);
-      const chargingStation = await ChargingStationStorage.getChargingStation(tenant.id, chargingStationId);
+      const chargingStation = await ChargingStationStorage.getChargingStation(tenant.id, Constants.UNKNOWN_STRING_ID, {}, null, chargingStationId);
```

Comment:
Nope, create a new method getChargingStationBySerialNumber(...)

Reply:
Ok, I changed it